### PR TITLE
Add 1D Profile viewer doc for IRIS Level 2 data sets

### PR DESCRIPTION
**Pull request by kakirastern**
## Description

Add documentation to explain how one can go about using `Glue`'s 1D Profile viewer in order to investigate a 4D stacked sequential raster cube using sliders provided as the Optional Widgets. Currently this feature is experimental as PR https://github.com/glue-viz/glue/pull/2156.

### CODE REVIEW

File: docs/guide-to-glue-1dprofile-viewer-for-iris-data.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
+.. _guide_to_1dprofile_viewer_for_iris_data:
+
+=========================================================================
+A Guide to using Glue's 1D Profile viewer to probe IRIS Level 2 data sets
+=========================================================================
+
+Loading multi-scan IRIS Level 2 date sets and stacking the sequential raster cubes
```
Comment (nabobalis) [inline docs/guide-to-glue-1dprofile-viewer-for-iris-data.rst]:
```suggestion
Loading multi-scan IRIS Level 2 data sets and stacking the sequential raster cubes
```

File: docs/guide-to-glue-1dprofile-viewer-for-iris-data.rst [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+.. image:: images/raster-and-sji-data-loaded-to-data-collection.png
+   :width: 800
+   :alt: Loaded raster and SJI data sets in the Data section of the Data Collection window
+
+Now we are ready to visualize the 2D slices of the imported data cubes.
+
+Using Glue's 2D Image viewer to get the indices and slices needed for the 1D spectrum plot
+------------------------------------------------------------------------------------------
+To plot 2D slices of the N-D data cubes, we will need to drag the data ("C_II_1336" in this
```
Comment (nabobalis) [inline docs/guide-to-glue-1dprofile-viewer-for-iris-data.rst]:
```suggestion
To plot 2D slices of the ND data cubes, we will need to drag the data ("C_II_1336" in this
```

File: docs/guide-to-glue-1dprofile-viewer-for-iris-data.rst [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+example) concerned from the Data Collection area and drop it at the large plotting window
```
Comment (nabobalis) [inline docs/guide-to-glue-1dprofile-viewer-for-iris-data.rst]:
```suggestion
example) concerned from the "Data Collection" area and drop it at the large plotting window
```

File: docs/guide-to-glue-1dprofile-viewer-for-iris-data.rst [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+   :width: 800
+   :alt: Dragging and dropping the same data to the viewing window to get the 1D Profile viewer
+
+.. image:: images/choosing-the-1d-profile-viewer.png
+   :width: 800
+   :alt: Choosing the 1D Profile viewer to generate 1D spectra of the sun
+
+Once the 1D Profile viewer shows up, you will be shown a collapsible version of the ``Maximum``
+function of the 1D spectrum. However, this may not be very telling and is not a genuine 1D
+spectrum, but is some statistical representation of the maximum of the quantity concerned.
```
Comment (nabobalis) [inline docs/guide-to-glue-1dprofile-viewer-for-iris-data.rst]:
```suggestion
spectrum, but is a statistical representation of the maximum of the quantity concerned.
```

**Review by nabobalis — APPROVED** [review]

**Comment by kakirastern** [comment]
Thanks @nabobalis! But I may need to update this doc later as @astrofrog has requested some changes to the corresponding PR https://github.com/glue-viz/glue/pull/2156.

**Comment by nabobalis** [comment]
Thats fine.

**Comment by kakirastern** [comment]
Thanks @nabobalis! I will be updating the doc before end of GSoC.

**Pull request merged**